app/boards/shields/breeze/breeze_right.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "breeze.dtsi"
+
+&default_transform {
+	row-offset = <5>;
+};
+
+&kscan0 {
+		row-gpios
+			= <&pro_micro 21  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro 20  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro 19  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro 18  (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			, <&pro_micro 15 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>
+			;
+};
```

Comment:
I see these are the same GPIO as the ones in the left .overlay. This can probably just be put in the .dtsi then instead of in each, right? The left side can just have the include line and on the right side the include and row offset.